scripts/link-data.sh

```diff
@@ -76,7 +76,7 @@ do
   
   # Declare the short_index directory and source directory
   index_dest=${index_dir}/short_index
-  index_source=${index_dir_names}
+  index_source=${index_source_names}
```

Comment:
This is going to always pick the first one, I think. Arrays in bash are strange, and you are trying to move through two at once, which is more challenging. I'd just make a separate step, & not try to use a loop for this.

Reply:
Fine with me! Done in ae7e1bc